cloudinit/config/cc_disk_setup.py

```diff
@@ -133,8 +134,9 @@ def handle(_name, cfg, cloud, log, _args):
             if not isinstance(definition, dict):
                 log.warning("Invalid disk definition for %s" % disk)
                 continue
-
             try:
+                if definition.get("timeout"):
+                    block_until_file_exists(disk, definition.timeout())
```

Comment:
are you *calling* timeout ?
i think you would want `block_until_file_exists(disk, definition.get('timeout'))`

```python

if definition.get("timeout"):
    utils.wait_for_files([disk], definition.get('timeout'), naplen=1)
```

Reply:
Yes I think you're correct whoops